src/pages/status/SetupSteps/ChequebookDeployFund.tsx

```diff
@@ -18,14 +24,11 @@ const ChequebookDeployFund = ({ isLoading, chequebookAddress, chequebookBalance
         {!(chequebookAddress?.chequebookaddress && chequebookBalance && chequebookBalance?.totalBalance > 0) && (
```

Comment:
It would be good to simplify this expression, possibly by extracting variables.

Also the optional chaining is not required because the `chequebookBalance` variable is checked before.

```suggestion
        {!(chequebookAddress?.chequebookaddress && chequebookBalance && chequebookBalance.totalBalance > 0) && (
```